_data/testimonials.yml

```diff
@@ -28,4 +28,4 @@
 
 - name: Maicol Battistini
   username: maicol07
-  text: Makeroid, a free platform to let your ideas become android apps. Simple, beatiful and powerful.
+  text: Makeroid, a free platform to let your ideas become android apps. Simple, beautiful and powerful.
```

Comment:
Should the `a` in `android` be capitalised?

Reply:
@conorshipp @barreeeiroo Fixed.